android/java/org/chromium/chrome/browser/vpn/timer/TimerDialogFragment.java

```diff
@@ -0,0 +1,83 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+package org.chromium.chrome.browser.vpn.timer;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment;
+
+import org.chromium.chrome.R;
+import org.chromium.chrome.browser.vpn.utils.BraveVpnProfileUtils;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class TimerDialogFragment
```

Comment:
doesn't presubmit ask to describe the class?

Reply:
I didn't see any complains about adding details for new class.